src/containers/projectcontainer.js

```diff
@@ -29,46 +35,67 @@ class ProjectContainer extends Component {
     }
   }
 
-
-  onProjectUpdate = (data) => {
-    if (this.props.project.data === null) {
+  onSubmit = (e) => {
+    e.preventDefault();
+    const data = this.props.form.data;
+    // TODO: Bytte til selected_project?
```

Comment:
Gets me every time :)
20. jun. 2016 13:44 skrev "Erlend Hamberg" <notifications@github.com>:

> In src/containers/projectcontainer.js
> <https://github.com/blankoslo/floq-projects/pull/3#discussion_r67676208>:
>
> > @@ -29,46 +35,67 @@ class ProjectContainer extends Component {
> >      }
> >    }
> >
> > -
> > -  onProjectUpdate = (data) => {
> > -    if (this.props.project.data === null) {
> > +  onSubmit = (e) => {
> > +    e.preventDefault();
> > +    const data = this.props.form.data;
> > +    // TODO: Bytte til selected_project?
>
> Anglo-Saxon please ;)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/blankoslo/floq-projects/pull/3/files/9e35bea115781469b82dafc6cea51b5136366cf9#r67676208>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABCBjBvULs2s2UJGQin2DZrJnKySyzJdks5qNn01gaJpZM4I5nYD>
> .
>